shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/BindingTableRule.java

```diff
@@ -80,7 +80,12 @@ public String getBindingActualTable(final String dataSource, final String logicT
         throw new ShardingSphereConfigurationException("Cannot find binding actual table, data source: %s, logic table: %s, other actual table: %s", dataSource, logicTable, otherActualTable);
     }
     
-    Collection<String> getAllLogicTables() {
+    /**
+     * Get list of logical table.
+     *
+     * @return logical tables.
+     */
```

Comment:
If this function is just for this PR, maybe we can remove it?

Reply:
@tristaZero 

There are three citations.